dmg/dmg.c

```diff
@@ -45,14 +45,24 @@ int main(int argc, char* argv[]) {
 	int opt;
 	char *cmd, *infile, *outfile;
 	char *key = NULL;
+	Compressor comp;
+	int ret;
 	
 	TestByteOrder();
+	getCompressor(&comp, NULL);
 
-	while ((opt = getopt(argc, argv, "k:")) != -1) {
+	while ((opt = getopt(argc, argv, "k:J:")) != -1) {
 		switch (opt) {
 			case 'k':
 				key = optarg;
 				break;
+			case 'J':
+				ret = getCompressor(&comp, optarg);
```

Comment:
Am I reading this correctly that this changes the default to _no_ compression? If so, please adjust this to maintain the current behaviour.

Reply:
No, we call `getCompressor(&comp, NULL)` above, and that uses bzip2 by default to preserve the existing behavior.

Reply:
Thanks, I clearly missed that detail of getCompressor!